### ISSUE TITLE
Vagrant only works on version 2.3.0

### ISSUE THREAD

**Issue by n0nag0n** [hashicorp/vagrant#13487]
### Debug output

https://gist.github.com/n0nag0n/d33e055a3f249c52c78ee9672f165e2d

### Expected behavior

Ok, so this will take a bit to explain so bear with me.

Ever since 2.3.1, each new version I work with suddenly doesn't boot. the vm and ends with a Virtualbox poweroff. If I revert back to 2.3.0 it is just fine. I can't figure out what changed between 2.3.0 and each subsequent version cause the diffs seem simple enough. I have tried what feels like everything. I have tried multiple computers running Linux Mint. I have tried wiping out both the `.vagrant/` and `~/.vagrant.d/` directories and starting from the beginning. I've tried wiping out the VMs, I've tried Virtualbox 6.0 and 6.1 with the same results. Everything I tried pointed back to only version 2.3.0 working. 

The latest straw that now has me feeling backed into a corner was I upgraded Linux Mint from 21.3 to 22. In doing so it is now Ubuntu 24.02 LTS. Virtualbox 6.1 doesn't support 24.02 so I have to go to 7.0. Vagrant 2.3.0 doesn't support 7.0 so I have to upgrade in order to make it work. I've tried installing VirtualBox via Software Manager, and from downloading from Virtualbox directly and installing via deb package. So I'm trying with 2.4.1 with no improvement into what's happening. So I can no longer use any of my vagrant environments because nothing will get brought up because I'm forced to use a newer version than 2.3.0. 

### Actual behavior

No matter what VM I try to bring up with virtualbox, it always ends up with the following error:

```
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'poweroff' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.

The primary issue for this error is that the provider you're using
is not properly configured. This is very rarely a Vagrant issue.
```

#### Vagrant version

Any version after 2.3.0.

#### Host operating system

Linux Mint 21.3 and 22

#### Guest operating system

Ubuntu 22.04 and generic/rocky8 and others.

#### Steps to reproduce

1. vagrant up
2. cry

#### Vagrantfile

```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "n0nag0n/firefly"
  config.vm.network "forwarded_port", guest: 80, host: 8888
  config.vm.synced_folder "./.", "/var/www/", mount_options: ["dmode=777,fmode=777"]
  config.vm.provision "shell", inline: <<-SHELL
	sed -i 's#DocumentRoot /var/www$#DocumentRoot /var/www/public#g' /etc/apache2/sites-available/000-default.conf
	systemctl restart apache2
  SHELL
end
```

**Comment by mlavi** [comment]
I have a similar issue with:
- LinuxMint 22
- VirtualBox Version 7.0.20 r163906 (Qt5.15.13)
- Vagrant 2.4.1
which worked fine today, but I recently added my user to the `vboxusers` group and now I see similar issues: difficulty creating VMs with vagrant, seeing VMs with the Oracle GUI that I've used `vagrant destroy --force`, etc.

**Comment by mlavi** [comment]
I have a similar issue with:
- LinuxMint 22
  - 6.8.0-39-generic #39-Ubuntu SMP PREEMPT_DYNAMIC Fri Jul  5 21:49:14 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
- VirtualBox Version 7.0.20 r163906 (Qt5.15.13) installed via https://www.virtualbox.org/wiki/Linux_Downloads instructions
- Vagrant 2.4.1
which worked fine today, but I most recently added my user to the `vboxusers` group and now I see similar issues: difficulty creating VMs with vagrant, mainly complaining about ports being in use, seeing VMs with the Oracle GUI that I've used `vagrant destroy --force`, etc.

I've removed myself from the vboxusers group:
`sudo usermod --remove --groups vboxusers ${SUDO_USER} && sudo reboot`
and verified with `id` and `vagrant up` proceeds further than before, but still cannot vagrant destroy properly, etc.

**Comment by n0nag0n** [comment]
I do have some other developers I work with that do not have this problem, and the one core difference that I can think of is that I work off of AMD based workstations and they work off of Intel. Not sure if that's worth mentioning.

**Comment by mlavi** [comment]
That could be a clue, I am on AMD, too. I'll amend the ticket, thanks!

```shell
$ sudo lshw -class cpu | head -n 4
  *-cpu                     
       description: CPU
       product: AMD Ryzen [omitted]
       vendor: Advanced Micro Devices [AMD]
```

**Comment by n0nag0n** [comment]
I read through some articles of people having issues with AMD processors and virtualbox, but nothing seemed to help. I decreased the cpus to 1 cpu and turned on the virtualbox gui and it's interesting that I see the gui, it shows that I can login and it stays that way for about 10 seconds, and then I get the poweroff message.

![image](https://github.com/user-attachments/assets/4009bc78-8fec-4890-a852-cca283c9bee7)

Here's the article I tried. https://ostechnix.com/how-to-increase-memory-and-cpu-on-vagrant-machine/#:~:text=To%20add%20memory%20and%20CPU%20to%20a%20Vagrant,%7Cv%7C%20v.memory%20%3D%202048%20v.cpus%20%3D%202%20end  The only equivalent setting I had in my BiOS is if Virtualization is enabled, and it is. Secure Boot is disabled.

I have a Dell Inspiron with a Ryzen 7 5700u AMD processor.

**Comment by n0nag0n** [comment]
Hot freakin' dog. I think I figured it out. 

There is something wrong with the build process for standalone binary builds (at least in my use case). In the following screenshot if you install via binary download, that's where I had the problem. 2.3.0 was the last one to work for me. I installed the repo locally for me on Mint, but had to overcome a problem and installed vagrant 2.4.1 that way and it worked!

![image](https://github.com/user-attachments/assets/75ef99a9-6b29-4325-bb51-0940f04562ba)

The problem I had to overcome was in my `/etc/apt/sources.list.d/hashicorp.conf` it had `https://apt.releases.hashicorp.com virginia main` and I needed to change `virginia` to `bookworm` and it worked!
